src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -0,0 +1,218 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AppBundle\Controller\Admin;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+use AppBundle\Entity\Post;
+
+/**
+ * Controller used to manage blog contents in the backend.
+ *
+ * Please note that the application backend is developed manually for learning
+ * purposes. However, in your real Symfony application you should use any of the
+ * existing bundles that let you generate ready-to-use backends without effort.
+ * See http://knpbundles.com/keyword/admin
+ *
+ * @Route("/admin/post")
+ * @Security("has_role('ROLE_ADMIN')")
+ *
+ * @author Ryan Weaver <weaverryan@gmail.com>
+ * @author Javier Eguiluz <javier.eguiluz@gmail.com>
+ */
+class BlogController extends Controller
+{
+    /**
+     * Lists all Post entities.
+     *
+     * This controller responds to two different routes with the same URL:
+     *   * 'admin_post_index' is the route with a name that follows the same
+     *     structure as the rest of the controllers of this class.
+     *   * 'admin_index' is a nice shortcut to the backend homepage. This allows
+     *     to create simpler links in the templates. Moreover, in the future we
+     *     could move this annotation to any other controller while maintaining
+     *     the route name and therefore, without breaking any existing link.
+     *
+     * @Route("/", name="admin_index")
+     * @Route("/", name="admin_post_index")
+     * @Method("GET")
+     */
+    public function indexAction()
+    {
+        $entityManager = $this->getDoctrine()->getManager();
+        $posts = $entityManager->getRepository('AppBundle:Post')->findAll();
+
+        return $this->render('admin/blog/index.html.twig', array('posts' => $posts));
+    }
+
+    /**
+     * Creates a new Post entity.
+     *
+     * @Route("/new", name="admin_post_new")
+     * @Method({"GET", "POST"})
+     *
+     * NOTE: the Method annotation is optional, but it's a recommended practice
+     * to constraint the HTTP methods each controller responds to (by default
+     * it responds to all methods).
+     */
```

Comment:
__[development][code_sniffer]__
* Missing @return tag in function comment

<!-- Comment by [SideCI](https://sideci.com) -->